charmhelpers/fetch/ubuntu.py

```diff
@@ -216,14 +218,26 @@ def filter_missing_packages(packages):
     )
 
 
-def apt_cache(in_memory=True, progress=None):
-    """Build and return an apt cache."""
-    from apt import apt_pkg
-    apt_pkg.init()
-    if in_memory:
-        apt_pkg.config.set("Dir::Cache::pkgcache", "")
-        apt_pkg.config.set("Dir::Cache::srcpkgcache", "")
-    return apt_pkg.Cache(progress)
+def apt_cache(*_):
```

Comment:
In case someone is calling this externally using keyword-style params, it might be worth handling those.  Something like:

```python
def apt_cache(*_, **__):
```

NB: Apparently, you can't reuse a single underscore as a placeholder name for multiple params in a function def.

Reply:
Good catch, I'll post a commit to address that just in case.